src/rss.rs

```diff
@@ -352,6 +352,14 @@ fn create_feed(conn: &rusqlite::Connection, feed: &Feed) -> Result<FeedId> {
     Ok(feed_id)
 }
 
+pub fn delete_feed(conn: &rusqlite::Connection, feed_id: FeedId) -> Result<()> {
+    let mut statement = conn.prepare("DELETE FROM feeds WHERE id = ?1")?;
+    statement.execute(params![feed_id])?;
+    let mut statement = conn.prepare("DELETE FROM entries WHERE feed_id = ?1")?;
+    statement.execute(params![feed_id])?;
+    Ok(())
+}
```

Comment:
Since this is a database mutation and the concepts (feed and entries) are linked, I think it makes sense to perform these actions as a single transaction (I think this is the relevant documentation https://docs.rs/rusqlite/latest/rusqlite/struct.Transaction.html).